### PR TITLE
Excepted URIs in RedirectIfAuthenticated middleware

### PR DESCRIPTION
**Pull request by olegmelnik**
Sometimes need for some routes from the group disable the redirect

### CODE REVIEW

**Comment by vlsoprun** [comment]
It may be better to use in_array?
```
return in_array($request->getRequestUri(), $this->except);
```

**Comment by olegmelnik** [comment]
@vlsoprun thanks!

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

File: app/Http/Middleware/RedirectIfAuthenticated.php [hunk 1]
```diff
@@ -17,10 +27,21 @@ class RedirectIfAuthenticated
...
             return redirect('/');
         }
 
         return $next($request);
     }
+
+    /**
+     * Checking request for excepted URI.
+     *
+     * @param Request $request
```
Comment (GrahamCampbell) [inline app/Http/Middleware/RedirectIfAuthenticated.php]: fully qualified name required please

Comment (GrahamCampbell) [inline app/Http/Middleware/RedirectIfAuthenticated.php]: and spacing needs correcting

**Comment by olegmelnik** [comment]
@GrahamCampbell I have removed the fork from my account and now I can not fix. I will recreate a fork again

**Pull request closed without merging**